_i18n/en/proton-c.md

```diff
@@ -36,7 +36,7 @@ You can find the Proton C for purchase at:
 * [Mykeyboard](https://mykeyboard.eu/catalogue/qmk-proton-c-rev-2_1246/) (EU)
 * [Daily Clack](https://dailyclack.com/products/qmk-proton-c) (AUS)
 
-Support for Split Keyboards, RGB Underglow, and Backlighting are being worked on and should be available soon.
+Split Keyboards, RGB Underglow, and Backlighting are all supported!
```

Comment:
Given <https://docs.qmk.fm/#/feature_split_keyboard> has 

> ARM is not yet fully supported for Split Keyboards and has many limitations. Progress is being made, but we have not yet reached 100% feature parity.

Maybe this should be worded to be less concrete.

```suggestion
ARM feature parity has not yet reached 100%, be sure to check the relevant hardware/software feature on <https://docs.qmk.fm/> for any current limitations.
```